examples/time_series/MvGaussianRandomWalk_demo.myst.md

```diff
@@ -226,6 +228,12 @@ ax.legend()
 ax.set_title("Posterior Predictive Samples and the Three Correlated Series");
 ```
 
+## Authors
+* updated to best practices by Lorenzon Itoniazzi in October, 2021 ([pymc-examples#195](https://github.com/pymc-devs/pymc-examples/pull/195))
+* updated to v5 by Chris Fonnesbeck in February, 2023 
+
++++
+
```

Comment:
Can you add the page footer at the bottom too? Commenting here because I can't do it at the last line